### PR TITLE
Changed "Report a bug" link in user dropdown menu

### LINKED ISSUE #3001: "Report Bugs" in the top right dropdown menu still links to Canny.
SHould be linking to: https://portal.productboard.com/colony/1-colony-portal/tabs/4-bugs

### PR DESCRIPTION
**Pull request by thycommodore**
## Description

- In the user menu (upper right, click avatar), changed the "Report a Bug" link to point to Productboard.

**Changes** 🏗

* "Report a Bug" link was updated to point to our active bug repo

Resolves #3001

### CODE REVIEW

File: src/modules/users/components/AvatarDropdown/AvatarDropdownPopover.tsx [hunk 1]
```diff
@@ -119,7 +119,7 @@ const AvatarDropdownPopover = ({
     <DropdownMenuSection separator>
       <DropdownMenuItem>
         <ExternalLink
-          href="https://colony.canny.io"
+          href="https://portal.productboard.com/colony/1-colony-portal/tabs/4-bugs"
```
Comment (arrenv) [inline src/modules/users/components/AvatarDropdown/AvatarDropdownPopover.tsx:122]: This is giving a lint error due to how long the URL is on the line.

You can get around this by setting the URL as a constant. For example:

long-urls

**Review by arrenv — CHANGES_REQUESTED** [review]
Thank you for making the update. I have just provided a comment regarding the linting error.

**Comment by rdig** [comment]
@arrenv maybe jump in and do this change yourself to get this merged faster ?

Unless @thycommodore wants to play around with our codebase some more :)

**Comment by arrenv** [comment]
@rdig yeah, happy to. Just thought I would give @thycommodore a chance to update, up to you Commodore, I can take over if you prefer.

**Comment by thycommodore** [comment]
@arrenv yes can you please resolve? Thank you for educating me!

**Comment by arrenv** [comment]
As requested, I have taken over this PR.

I have moved the URL into a constants file and pulled from that, this is probably something we could do more of where possible if mutually agreed. I will raise during our focus on maintenance in the next quarter.

File: src/modules/constants.ts [hunk 2]
```diff
@@ -165,3 +165,6 @@ export const TOKEN_LOGOS_REPO_URL = `https://raw.githubusercontent.com/trustwall
 export const NETWORK_RELEASES_URL = `https://github.com/JoinColony/colonyNetwork/releases/tag`;
 
 export const SMALL_TOKEN_AMOUNT_FORMAT = '0.00000...';
+
+export const FEEDBACK_LINK =
```
Comment (rdig) [inline src/modules/constants.ts:169]: We usually keep links like these inside the component itself, but It's ok for it being in here as well.

**Review by rdig — APPROVED** [review]
Good to go!

**Review by arrenv — APPROVED** [review]

**Pull request merged**